NVIDIA/benchmarks/unet3d/implementations/mxnet/runtime/arguments.py

```diff
@@ -90,4 +90,16 @@
 PARSER.add_argument('--use_cached_loader', '-ucl', dest='use_cached_loader', action='store_true', default=False)
 PARSER.add_argument('--stick_to_shard', '-sts', dest='stick_to_shard', action='store_true', default=False)
 
-PARSER.add_argument('--use_nvshmem', dest='use_nvshmem', action='store_true', default=False)
\ No newline at end of file
+PARSER.add_argument('--use_nvshmem', dest='use_nvshmem', action='store_true', default=False)
```

Comment:
I don't believe these additional parser arguments belong here.  These look like arguments from our BERT benchmark code.  None of these arguments are used by the code here, and wouldn't do anything if you tried to enable them.